lib/interpreters/compute_several_solutions.rb

```diff
@@ -18,6 +18,16 @@
 
 module Interpreters
   class SeveralSolutions
+    def self.duplicate_service_vrp(service_vrp, vrp_hash = nil)
+      service_vrp.map{ |key, value|
+        if key == :vrp
+          vrp_hash ||= value.as_json
+          value = Models::Vrp.create(vrp_hash)
+        end
+        [key, value]
+      }.to_h
```

Comment:
I find this more expressive because of explicit `dup` and override of `[:vrp]`, what do you think?
```suggestion
      vrp_hash ||= service_vrp[:vrp].as_json
      service_vrp.dup.tap{ |hash| hash[:vrp] = Models::Vrp.create(vrp_hash) }
```